sample/kotlin/src/main/sqldelight/com.datadog.android.sample.data.db/Logs.sq

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS logs (
```

Comment:
Is this file supposed to be in the `main` source set ? Is it linked with SQLDelight?

Reply:
yeah...they specifically wanted there, I tried to move it in the flavour source folder but then at compile time they are not able to find the generated class. I might try a different trick there actually but if that doesn't work I am afraid I will have to keep it in the main.